package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "awesome-lint",
-	"version": "0.8.0",
+	"version": "0.9.0",
```

Comment:
Don't bump version in a pull request. It's up to the maintainers when, how, and what version to bump :)

Reply:
Ah, my apologies, that makes sense.